.circleci/config.yml

```diff
@@ -0,0 +1,109 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/build
+    environment:
+      PIPENV_VENV_IN_PROJECT: TRUE
+    docker:
+      - image: circleci/python:3.7.0-stretch-node
+    steps:
+      - checkout 
+      - run:
+          name: Update npm
+          command: sudo npm install -g npm@latest 
+      - run:
+          name: Update Python Installers 
+          command: sudo python -m pip install -U setuptools pip pipenv
+      - run:
+          name: Install Python Libraries 
+          command: pipenv run pip install -r scripts/requirements.txt
+      - run:
+          name: Install npm dependencies 
+          command: npm install
+      - run:
+          name: Set version 
+          command: pipenv run auto_version --config=scripts/auto_version.toml --bump=patch --news
+      - run: 
+          name: Build nodes
+          command: npm run gulp 
+  news:
+    working_directory: ~/build 
+    docker:
+      - image: circleci/python:3.7.0-stretch-node
+    steps:
+      - checkout
+      - run:
+          name: Update Python Installers
+          command: sudo python -m pip install -U setuptools pip pipenv 
+      - run:
+          name: Install Python Libraries
+          command: pipenv run pip install -r scripts/requirements.txt
+      - run:
+          name: Set version 
+          command: pipenv run auto_version --config=scripts/auto_version.toml --bump=patch --news
+      - run:
+          name: Update changelog
+          command: pipenv run towncrier --yes --name="" --version=$(cd../../ && auto_version --config=scripts/auto_version.toml)
+          working_directory: docs/news
+      - store_artifacts:
+          path: CHANGELOG.md
+  tpip:
+    working_directory: ~/build
+    docker:
+      - image: circleci/python:3.7.0-stretch-node-browsers
+    steps:
+      - checkout
+      - run:
+          name: Update npm
+          command: sudo npm install -g npm@latest
+      - run:
+          name: Install npm dependencies
+          command: npm install
+      - run:
+          name: Build Third Party Intellectual Property file 
+          command: npm run tpip
+      - store_artifacts:
+          path: tpip.csv 
+  publish:
+    working_directory: ~/build
+    environment:
+      PIPENV_VENV_IN_PROJECT: TRUE
+    docker: 
+      - image: circleci/python:3.7.0-stretch-node-browsers
+    steps:
+      - checkout 
+      - attach_workspace:
+          at: ./
+      - run:
```

Comment:
You need to run gulp again before you release, otherwise the package won't have any built nodes in it